src/common/contactsFunctionality/ContactUtils.ts

```diff
@@ -104,6 +104,15 @@ export function getSocialUrl(contactId: ContactSocialId): string {
 
 			case ContactSocialType.LINKED_IN:
 				socialUrlType = "linkedin.com/in/"
+				break
+
+			case ContactSocialType.MASTADON:
+				socialUrlType = "mastodon.social/@"
+				break
+
+			case ContactSocialType.BLUE_SKY:
+				socialUrlType = "bsky.app/profile/"
```

Comment:
Bluesky is also federated, I don't know how it works for them normally, maybe @wrdhub knows how user handles work